### ISSUE TITLE
No debug/log error 

### ISSUE THREAD

**Issue by surfzoid** [uraimo/run-on-arch-action#99]
Hi,
My workflow have error, but action/dock just display :exit with error 1
https://github.com/surfzoid/QtVsPlayer/actions/runs/3263386857/jobs/5362113788

**Comment by uraimo** [comment]
From the look of it, it just seems the result of multiple yaml formatting issue, the action can't even load the the run section correctly. Will take a look but could be impossible to intercept from within the action.

**Comment by surfzoid** [comment]
yep, yaml and python suck to auto indent, but seem strange anyway

**Comment by surfzoid** [comment]
i see line 167 and 168 formating bad, but i adjusted it in next version, i also add set -x, but alway no output
